champtc/opencti
# fix: adding the notify checkbox for the new analysis modal

Added a checkbox in the new analysis modal that will include a boolean value with the payload when submitting a new analysis request. There is also some code formatting. 

It is important to note that this **was not** a feature before. It was only defined for creating a new scan, not analysis. Therefor this is labeled as an enhancement. 

**Testing**
* Navigate to the Vulnerability Assessment page
  * Make sure you are in an org with an analysis (example)
* Click on the dropdown for an existing analysis
* Select 'New Analysis` from the menu
* Fill in the form, checking the checkbox to receive an email
* Submit
